build-aux/io.github.flattool.Ignition.json

```diff
@@ -0,0 +1,65 @@
+{
+    "id" : "io.github.flattool.Ignition",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.node20"
+    ],
+    "command" : "io.github.flattool.Ignition",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=xdg-config/autostart:create",
+        "--filesystem=xdg-data/icons:ro",
+        "--filesystem=xdg-data/flatpak:ro",
+        "--filesystem=/var/lib/flatpak:ro",
+        "--filesystem=/var/lib/snapd:ro",
+        "--filesystem=/snap:ro",
+        "--filesystem=host-os:ro"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "blueprint-compiler",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                    "tag" : "v0.14.0"
+                }
+            ],
+            "cleanup" : [
+                "*"
+            ]
+        },
+        {
+            "name" : "ignition",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [ "-Dprofile=development" ],
```

Comment:
I don't think this is correct.

Reply:
Ah nice catch. My apologies, I failed to switch the profile from devel to default. I've pushed a fix to this PR.